package.json

```
@@ -1,12 +1,13 @@
 {
-  "name": "portail-usager-conseille-numerique",
+  "name": "portail-usager-conseiller-numerique",
   "version": "0.0.0",
   "description": "Portail usager - Conseiller Numérique",
   "author": "Romain Cambonie <romain.cambonie@gmail.com>",
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start:prod": "ng serve --configuration production",
+    "start:dev": "ng serve --configuration development",
```

```suggestion
    "start": "ng serve --configuration development",
```
Pour éviter de changer la comdande de base

---

je préfère être explicite car de base ng serve target la configuration par defaut qui est production.